scripts/hello-world-deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "-> Updating system"
+yes | pacman -Syu
+
+if ! ( which npm )
```

Comment:
If we're doing "desired state" instead of "dependency checking" then you want all your conditionals to be in the positive as in "npm should be installed" -> then echo "installed", if not, else "install npm". There should be no actions in the "then" part of your conditionals. This is important and we can discuss why, but it involves idempotency and is HUGE at scale and quality.